dxr/plugins/clang/menus.py

```diff
@@ -118,31 +118,40 @@ def _more_menu_items(self, (macro_name,)):
 class TypeRef(_QualnameRef):
     @classmethod
     def _condensed_menu_data(cls, tree, prop):
-        """Hang onto qualname *and* kind."""
         return (super(TypeRef, cls)._condensed_menu_data(tree, prop) +
-                (prop.get('kind', ''),))
+                (prop.get('kind', ''), 'has_subclass' in prop, 'has_base_class' in prop))
```

Comment:
For example, we could default to 'type' or something here, add a case to kind_plurals, and have done with it.